openhands/runtime/impl/action_execution/action_execution_client.py

```diff
@@ -114,11 +119,6 @@ def _send_action_server_request(
         """
         return send_request(self.session, method, url, **kwargs)
 
-    @retry(
```

Comment:
Is this change intentional?

Reply:
Yes, because check_if_alive uses `_send_action_server_request` to send request, and in this PR, we already do retry for `_send_action_server_request`, so no need to do it again in `check_if_alive`